content/guides/getting-started/creating-a-cy-mount-command.md

```diff
@@ -2,4 +2,143 @@
 title: Creating a cy.mount Command
 ---
 
-CONTENT TBD
+We recommend creating a custom `cy.mount` command which wraps the mount command
+from the framework-specific libraries in your component tests. Doing so offers a
+few advantages:
+
+- You don't need to import the mount command into every test as the `cy.mount`
+  command is available globally
+- You can set up common scenarios that you usually have to do in each test, like
```

Comment:
I feel like the first time we mention the following things, they should be linked:

- Custom Command -> https://docs.cypress.io/api/cypress-api/custom-commands
- React Provider -> https://reactjs.org/docs/context.html
- Vue Plugins -> https://vuejs.org/v2/guide/plugins.html
- TypeScript -> https://docs.cypress.io/guides/tooling/typescript-support